config/waagent.conf

```diff
@@ -62,6 +62,9 @@ ResourceDisk.MountOptions=None
 # Enable verbose logging (y|n)
 Logs.Verbose=n
 
+# Enable Console logging, default is y
```

Comment:
we have conf files per distro... you may want to add this to config/*/waagent.conf

btw - why comment it out if the default is true anyway?

Reply:
Thank you for reviewing!

I commented it out because there are other config which are enabled by default such as AutoUpdate.Enabled.
Should I uncomment it?

Anyway I'm going to add this to distro confs.